.github/workflows/release.yaml

```diff
@@ -8,14 +8,17 @@ on:
 jobs:
   integration-test:
     uses: ./.github/workflows/ci.yaml
-        
+
   publish-bundle:
     name: Publish bundle
     runs-on: ubuntu-latest
     needs:
       - integration-test
     env:
       CHARMCRAFT_AUTH: ${{ secrets.CHARMHUB_TOKEN }}
+    strategy:
+      matrix:
+        bundle-name: [postgresql-bundle]
```

Comment:
We can probably remove the matrix now as there should be only one bundle (with TLS enabled).